src/app/pages/LiveRadioPage/index.jsx

```diff
@@ -1,13 +1,14 @@
 import React, { useContext } from 'react';
-import { string, shape, object, arrayOf } from 'prop-types';
+import { string, shape, object } from 'prop-types';
 import styled from 'styled-components';
-import path from 'ramda/src/path';
+import { Headline } from '@bbc/psammead-headings';
+import Paragraph from '@bbc/psammead-paragraph';
 import ATIAnalytics from '../../containers/ATIAnalytics';
 import MetadataContainer from '../../containers/Metadata';
 import ChartbeatAnalytics from '../../containers/ChartbeatAnalytics';
 import Grid, { GelPageGrid } from '#app/components/Grid';
 import LinkedData from '../../containers/LinkedData';
-import RadioPageBlocks from '../../containers/RadioPageBlocks';
+import AudioPlayer from '#containers/RadioPageBlocks/Blocks/AudioPlayer';
```

Comment:
Have we considered also renaming the RadioPageBlocks container?

Reply:
Yes, that's the focus of the next PR. We wanted to remain within the 1 hour timebox.